src/_base/helm/app/values.yaml.twig

```diff
@@ -23,3 +23,4 @@ resources:
 {% include blocks ~ 'resources.yml.twig' %}
 
 ingress: "standard" # standard or istio
+prefix: {{ @('workspace.name') ~ '-' }}
```

Comment:
I'd suggest moving this to an attribute and only overriding in the project in question